_data-prepper/pipelines/configuration/processors/key-value.md

```diff
@@ -15,12 +15,12 @@ Takes in a field and parses it into key/value pairs.
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
 source | No | String | The key in the event that is parsed. Default value is `message`.
-destination | No | String | The key where to output the parsed source to. Doing this overwrites the value of the key if it exists. Default value is `parsed_message`
+destination | No | String | The key where the output is parsed source to. Outing the parsed source overwrites the value of the key if it already exists. Default value is `parsed_message`
```

Comment:
This might just be my lack of familiarity with data prepper, but in case it's a typo I wanted to point it out. Is "outing" a thing or did you mean "outputting?" (I'm not familiar with data prepper so I might sound like a noob)

Reply:
Good catch - this is a typo.